examples/210-embedding_models.ipynb

```diff
@@ -77,6 +77,7 @@
      "name": "stderr",
```

Comment:
<p>May be we should point out that it could be useful to do before saving / after loading.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/etna-team/etna/pull/396/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
No matter, I think